components/massmint/modals/MintingModal.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <NeoModal
+    v-model="isModalActive"
+    scroll="clip"
+    :can-cancel="!loading"
+    @close="emit('close')">
+    <div class="p-7">
+      <div
+        v-if="loading"
+        class="is-flex is-flex-direction-column is-align-items-center">
+        <img src="/preloader.svg" width="200" />
+        <span class="mt-6">{{ $t('massmint.mintingYourNFTs') }}</span>
+      </div>
+      <div
+        v-else
+        class="is-flex is-flex-direction-column is-align-items-center">
+        <NeoIcon icon="circle-check" class="check-icon has-text-k-green" />
+        <span class="mt-4">{{ $t('massmint.mintDone') }}</span>
+        <div class="is-flex w-full is-justify-content-center pt-5 px-7">
+          <NeoButton
+            :label="$t('massmint.done')"
+            variant="k-accent"
+            class="is-flex is-flex-grow-1"
+            @click.native="emit('close')" />
+        </div>
+      </div>
+    </div>
+  </NeoModal>
+</template>
+
+<script setup lang="ts">
+import { NeoButton, NeoIcon, NeoModal } from '@kodadot1/brick'
+
+const props = defineProps<{
+  value: boolean
+  loading: boolean
+}>()
+
+const isModalActive = useVModel(props, 'value')
+
+const emit = defineEmits(['close'])
+</script>
+
+<style lang="scss" scoped>
+.check-icon {
```

Comment:
maybe check `NeoIcon` size prop (`large`)

Reply:
that prop doesn't work

Reply:
Aight, will open an issue